packages/components/src/Tooltip/src/Tooltip.tsx

```diff
@@ -0,0 +1,136 @@
+import {
+    useColorSchemeContext,
+    useStyledSystem,
+    type StyledComponentProps,
+    type StyledSystemProps
+} from "@hopper-ui/styled-system";
+import clsx from "clsx";
+import { forwardRef, useContext, type ForwardedRef } from "react";
+import { composeRenderProps, Tooltip as RACTooltip, useContextProps, type TooltipProps as RACTooltipProps } from "react-aria-components";
+
+import { HopperProvider } from "../../HopperProvider/index.ts";
+import { TextContext } from "../../typography/index.ts";
+import { composeClassnameRenderProps, cssModule, ensureTextWrapper, SlotProvider, type BaseComponentDOMProps } from "../../utils/index.ts";
+
+import { TooltipContext } from "./TooltipContext.ts";
+import { TooltipTriggerContext } from "./TooltipTriggerContext.ts";
+
+
+import styles from "./Tooltip.module.css";
+
+export const GlobalTooltipCssSelector = "hop-Tooltip";
+
+type PropsToOmit = "triggerRef" | "UNSTABLE_portalContainer" | "placement" | "containerPadding" | "offset" | "crossOffset" |
```

Comment:
[Nit] You had to omit these because we use them in the TooltipTrigger right? Could we use the interface from that component instead of repeating the same list?

Reply:
hmm not really, we still want some of the props. This is the way spectrum did it.